spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,17 @@
+Fabricator(:user) do
+  first_name {Faker::Name.first_name}
+  last_name {Faker::Name.last_name}
+  email {Faker::Internet.email}
+  password {"password"}
+  password_confirmation {"password"}
+end
+
+Fabricator(:user2) do
```

Comment:
I wouldn't name a fabricator `user2`. It appears you want this to be an invalid user. I would name in accordingly. You can even inherit from `user`.

```ruby
Fabricator(:invalid_user, from: :user) do
  password: nil
  password_confirmation: nil
end
```